[Compute RN][Maxwell Update 2] PCEE

rcaballeromx opened the pull request:
Release Notes PR for Prisma Cloud Enterprise Edition (SaaS) Maxwell Update 2

Commit:
[Compute RN][Maxwell Update 2] PCEE

divyabhushan-okta commented on cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc:
```diff
@@ -55,6 +55,11 @@ With this fix, the correct AWS product ARN for US and China regions are used.
 //added Kepler Update 2
 |The userinput:[--tarball] option in twistcli does not scan for compliance checks. Currently, only vulnerabilities are detected successfully.
 
+//CWP-47729
```
@rcaballeromx, I would write the latest blurbs on the top like I rearranged in [22.12 Known Issues](https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-12/prisma-cloud-compute-edition-release-notes/release-information/known-issues-22-12).
This will be easier for the user rather than scrolling down all the way to the end for the latest fixes.
WDYT?
Cc: @Pubs-MV

Pubs-MV replied:
correct. We want the latest fixes on the top. But the rest of the known issues are chronological.

divyabhushan-okta replied:
@Pubs-MV Moved.

ssugandh committed:
Update features-introduced-in-compute-june-2023.adoc

ssugandh committed:
Update features-introduced-in-compute-june-2023.adoc

ssugandh committed:
Update features-introduced-in-compute-june-2023.adoc

Commit:
[Compute RN][MaxwellUpdate2: CWP-34539]Add blurb

Commit:
[Compute RN][MaxwellUpdate2: CWP-44813]: Add blurb

Commit:
[Compute RN][MaxwellUpdate2]: Update CWP-47766

ssugandh committed:
Update features-introduced-in-compute-june-2023.adoc

ssugandh committed:
Keeping Breaking API Changes for now

Will remove if the decision is made to revert the change finally. Waiting to hear from Elvira.

divyabhushan-okta commented on cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-june-2023.adoc:
```diff
@@ -0,0 +1,164 @@
+2+|*Enhancements*
+
+//CWP-34539
+|*Support custom compliance checks*
```
This feature might not make it to Maxwell Update2, waiting to hear back from Elvira to confirm!

divyabhushan-okta replied:
Elvira confirmed, this feature is in for Maxwell Update2. Resolving the comment.

Commit:
[Compute RN][MaxwellUpdate2]:Move the latest fix at the top

Commit:
[Compute/CSPM RN]: Add blurb for CWP-45046

Commit:
[Compute/CSPM RN]: Update blurb CWP-44813

Commit:
[Compute/CSPM]: Update eos notification for CWP-43837

Commit:
[Comute/CSPM RN]: Grammar fix

ssugandh committed:
CWP-49167: Added deprecation notice for API

ssugandh committed:
Update features-introduced-in-compute-june-2023.adoc

Commit:
[CSPM/Compute RN]: Add deprecation notice for CWP-49167

Commit:
[CSPM/Compute RN]: Aggregate Deprecation section for API and UI+Update TOC xref

Commit:
[CSPM/Compute RN]: Add toc for Breaking Changes in API

Commit:
Add CVE Coverage and build number

Commit:
Merge branch 'maxwell-update2-rn-pcee' of https://github.com/PaloAltoNetworks/prisma-cloud-docs into maxwell-update2-rn-pcee

Commit:
Update current version and fix table

Commit:
Cosmetic fix

Commit:
Remove heading for CWP-47766

Commit:
Add links to Admin Guide
